sycl/doc/syclcompat/README.md

```diff
@@ -72,6 +72,40 @@ This document presents the public API under the [Features](#features) section,
 and provides a working [Sample code](#sample-code) using this library. Refer to
 those to learn to use the library.
 
+## Versioning
+
+SYCLcompat adopts [semantic versioning](https://semver.org/)
+(`major.minor.patch`) in a manner which aligns with oneAPI releases. Each oneAPI
+product release has an associated SYCLcompat release. When breaking changes have
+been made to SYCLcompat's API since the last release, `major` is incremented,
+otherwise `minor` is incremented. Use of `patch` is described below.
+
+Between release cycles, ongoing updates to SYCLcompat (including possibly
+breaking changes) are merged into DPC++'s
+[`sycl`](https://github.com/intel/llvm/tree/sycl) branch. To enable projects
+pulling SYCLcompat from source to distinguish from the latest release version,
+the `patch` version in the `sycl` branch is incremented immediately following a
```

Comment:
I think we should increase the `major` or `minor` version, not the patch, after every release.

If we increase the patch, tags are more complicated, and in case we needed to submit a patch for an already existing release version", there would be conflict between the released compiler version and the github repository version